apps/omg/lib/omg/alert/alarm.ex

```diff
@@ -12,17 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-defmodule OMG.API.Alert.Alarm do
+defmodule OMG.Alert.Alarm do
   @moduledoc """
   Interface for raising and clearing alarms.
   """
-  alias OMG.API.Alert.AlarmHandler
-  @behaviour AlarmHandler
+  alias OMG.Alert.AlarmHandler
 
-  @impl AlarmHandler
   def ethereum_client_connection_issue(node, reporter),
     do: {:ethereum_client_connection, %{node: node, reporter: reporter}}
 
+  def boot_in_progress(node, reporter),
+    do: {:boot_in_progress, %{node: node, reporter: reporter}}
+
   @spec raise({atom(), node(), module()}) :: :ok | :duplicate
```

Comment:
The highlighting in github is incorrect. 
- `Raise` is reserved in function body.
- `node` would warn you if it wasn't passed as argument, because compiler would try to resolve it as node().

Let me know if you still want to me to change it though.

Reply:
I don't have a strong opinion here, if it highlights incorrectly only in gh, then probably a non-issue.